docs/non-functional-requirements/internationalization.md

```diff
@@ -0,0 +1,31 @@
+# Internationalization and Localization
+
+Internationalization (i18n) and Localization (l10n) refer to the design and adaptation of software systems to support multiple languages, cultures, and regions, ensuring usability and compliance with local preferences and regulations.
+
+## Characteristics
+
+### Main characteristics of Internationalization
+
+- Text Externalization: Moving all user-facing text to external resource files to facilitate easy translation.
+- Unicode Support: Using Unicode or another character encoding that supports all necessary scripts and characters.
+- Date and Time Formatting: Designing the system to handle various date and time formats.
```

Comment:
This is also something which need to be careful with when using times to always use UTC on required things. Otherwise the time will be the time of the machine and it might cause a lot of confusion in different regions

Reply:
Good point.